component/frontend/Helper/Breadcrumbs.php

```diff
@@ -156,7 +156,7 @@ public static function addRelease($id, $name)
 				$Itemid = empty($Itemid) ? '' : '&Itemid=' . $Itemid;
 
 				$rootName = $name;
-				$rootURI = JRoute::_('index.php?option=com_ars&view=release&id=' . $id . $Itemid);
+				$rootURI = JRoute::_('index.php?option=com_ars&view=Releases&category_id=' . $id . $Itemid);
```

Comment:
This should be `view=Items&release_id` because we're displaying the Items view that lists all download items in a single release (you could say it's the "release" page but we chose to keep the plural views in the new ARS for consistency). The parameter it gets is the release ID, not the category ID.

If you push a commit with this change to your PR I can merge it :)